server/src/main/resources/db/changelog/changelog-testing.xml

```diff
@@ -2028,7 +2028,7 @@
     </changeSet>
     <changeSet author="awood" id="1413225753032-290">
         <comment>
-            Add the default quartz lock columns.
+             Add the default quartz lock columns. 
```

Comment:
I think this is a spurious change.  I've seen it pop up when I run the liquibase task though so I don't have a problem leaving it in.

Reply:
Let's leave it as is so that it doesn't keep showing up as a change.